tracks/rust/exercises/sublist/mentoring.md

```diff
@@ -12,6 +12,7 @@ A reasonable solution should do the following:
   a larger list
 - Utilize Eq for slices
 - not reimplement the functionality of the `windows` method
+- some student call 2 time the windows function instead of checking the size. For the performance point of view it's the same because the windows function do the check.
```

Comment:
Please rewrite for grammar / clarity.

Reply:
Is it better?

Reply:
Not really. Reading that text, I still don't understand the following points:

- Is this point recommending one technique over the other?
- If it is recommending one, which is it recommending?
- How is an additional `windows` call equivalent to a `len` call? Naively, I'd expect `O(n)` in the former case, and `O(1)` in the latter.